pkg/download/entities/entities.go

```diff
@@ -68,7 +69,7 @@ func (d *Downloader) Download(specificEntitiesTypes []string, projectName string
 	log.Debug("Fetching specific entities types to download")
 
 	// get ALL entities types
-	entitiesTypes, err := d.client.ListEntitiesTypes()
+	entitiesTypes, err := d.client.ListEntitiesTypes(context.TODO()) //TODO: real context
```

Comment:
Not included in any followup PR

Reply:
Thanks! I'll address the context TODOs I've missed in https://github.com/Dynatrace/dynatrace-configuration-as-code/pull/1055 as the all related to download or delete usage!